### PR TITLE
Ordinals readme

### PR DESCRIPTION
**Pull request by jrwbabylonlab**
image

### CODE REVIEW

**Review by gitferry — APPROVED** [review]

**Review by gitferry — COMMENTED** [review]
The title seems funny

File: docs/ordinals.md [hunk 1]
```diff
@@ -1,41 +1,58 @@
-# Ordinals in API system
+# Ordinals in API System
```
Comment (vitsalis) [inline docs/ordinals.md:1]:
```suggestion
# Detection of Inscriptions
```

File: docs/ordinals.md [hunk 2]
```diff
@@ -1,41 +1,58 @@
...
-This is accomplished through a connection to the  
-[Ordinal Service](https://github.com/ordinals/ord) and  
-a connection to the Unisat API.  
-Due to Unisat being a payed service and applying rate limits,  
-the API initially tries to get the status of a UTXO through the Ordinals Service,  
-and if that fails, then contacts the Unisat API,  
-effectively using it as a back-up mechanism to handle downtime from the Ordinals Service.  
+The Babylon Staking API provides optional endpoints to check whether a UTXO 
+contains an inscription. This functionality helps staking applications avoid 
+spending specific UTXOs. The API connects to the
```
Comment (vitsalis) [inline docs/ordinals.md:5]:
```suggestion
spending specific UTXOs that might hold inscriptions. The API connects to the 
```

File: docs/ordinals.md [hunk 3]
```diff
@@ -1,41 +1,58 @@
...
-the API initially tries to get the status of a UTXO through the Ordinals Service,  
-and if that fails, then contacts the Unisat API,  
-effectively using it as a back-up mechanism to handle downtime from the Ordinals Service.  
+The Babylon Staking API provides optional endpoints to check whether a UTXO 
+contains an inscription. This functionality helps staking applications avoid 
+spending specific UTXOs. The API connects to the 
+[Ordinal Service](https://github.com/ordinals/ord) and the Unisat API. Since 
+Unisat is a paid service with rate limits, the API first attempts to get the 
+UTXO status through the Ordinals Service. If that fails, it contacts the Unisat 
+API as a backup to handle Ordinals Service downtime.
```
Comment (vitsalis) [inline docs/ordinals.md:9]:
```suggestion
API as a backup to handle Ordinals Service downtime.

**Note**: This is an approximate solution and in no way it shall be considered
a solution that does not have false positives or negatives.
If someone intends to use this service to detect inscriptions,
they should do it assuming that the service will not return entirely accurate results,
and they should implement additional fail safe mechanisms for inscription detection.
```

**Review by vitsalis — APPROVED** [review]
some comments

**Pull request merged**